Core/src/main/java/com/paypal/android/core/graphql/common/Utils.kt

```diff
@@ -0,0 +1,18 @@
+package com.paypal.android.core.graphql.common
+
+import org.json.JSONArray
+import org.json.JSONObject
+
+fun JSONArray?.toStringsList(): List<String> {
+    return if (this == null) emptyList()
+    else {
+        MutableList(this.length()) {
+            this.getString(it)
+        }
+    }
+}
```

Comment:
nitpick, this could be a single expression function
```suggestion
fun JSONArray?.toStringsList(): List<String> = if (this == null) emptyList()
    else {
        MutableList(this.length()) {
            this.getString(it)
        }
    }
```